dvc/commands/gc.py

```diff
@@ -104,6 +107,17 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Keep data files for all Git commits.",
     )
+    gc_parser.add_argument(
+        "--time",
+        type=str,
+        dest="commit_time",
+        metavar="<commit_time>",
+        default=None,
+        help=(
+            "Keep experiments from the commits after(include) a certain time."
+            "The time format is '%Y-%m-%d'"
```

Comment:
```suggestion
            "The time format is '%%Y-%%m-%%d'"
```

Currently getting an error:

```bash
$ dvc gc --help
ERROR: unexpected error - unsupported format character 'Y' (0x59) at index 85
```